rustsec/tests/database.rs

```diff
@@ -27,31 +22,13 @@ fn vulnerabilities_default() {
     assert_eq!(vuln.len(), 1);
 }
 
-/// all package scope should be default
 #[test]
-fn query_vulnerabilities_default() {
+fn query_vulnerabilitie() {
     let lockfile_path = Path::new("./tests/support/cratesio_cargo.lock");
     let lockfile =
         Lockfile::load(lockfile_path).expect("Should find the lock file in support folder.");
     let db = DEFAULT_DATABASE.lock().unwrap();
-    let vuln_all =
-        db.query_vulnerabilities(&lockfile, &Query::crate_scope(), scope::Package::default());
+    let vuln_all = db.query_vulnerabilities(&lockfile, &Query::crate_scope());
     let vuln = db.vulnerabilities(&lockfile);
     assert_eq!(vuln_all, vuln);
 }
-
-/// packages without source should not be queried in `package::Scope::LocalCrates` but in `PackageScope::PublicCrates`
-#[test]
-fn query_vulnerabilities_scope_public() {
-    let lockfile_path = Path::new("./tests/support/local_cargo.lock");
-    let lockfile =
-        Lockfile::load(lockfile_path).expect("Should find the lock file in support folder.");
-    let db = DEFAULT_DATABASE.lock().unwrap();
-
-    let vuln_public =
-        db.query_vulnerabilities(&lockfile, &Query::crate_scope(), scope::Registry::Public);
-    assert_eq!(vuln_public.len(), 0);
-
-    let vuln_all = db.query_vulnerabilities(&lockfile, &Query::crate_scope(), scope::Registry::All);
-    assert_eq!(vuln_all.len(), 1);
-}
```

Comment:
This is the only functionality provided by the previous `db::scope` system.

I'll add replacement test(s) to exercise the registry scoping rules, testing that an advisory doesn't match if it's for a `registry` different from `source`.

Reply:
SGTM